markdown/bi-tools/DBeaver.md

```diff
@@ -0,0 +1,43 @@
+### DBeaver
+
+Before proceeding, ensure you have [configured your environment](../setup/configuration.md).
+
+Download the latest version of [DBeaver](https://dbeaver.io/ to use the Driver in DBeaver.
+
+**For Mac you must launch your BI tool through terminal to have the environment variables loaded.**
```

Comment:
Need a reference or inline -  how to launch DBeaver from a command line and how to specify environment variable.

Reply:
Do we care about environment variables loaded? We removed the dependency on this.